Core/src/Utils/DeformationGraph.cpp

```diff
@@ -169,13 +174,13 @@ void DeformationGraph::setPosesSeq(std::vector<unsigned long long int> * poseTim
 
         imin = std::min(imin, (int)sampledGraphTimes.size() - 1);
 
-        if(abs(sampledGraphTimes[imin] - poseTime) <= abs(sampledGraphTimes[imid] - poseTime) &&
-           abs(sampledGraphTimes[imin] - poseTime) <= abs(sampledGraphTimes[imax] - poseTime))
+        if(ulonglongDiffAbs(sampledGraphTimes[imin],poseTime) <= ulonglongDiffAbs(sampledGraphTimes[imid],poseTime) &&
```

Comment:
Time comes from tick which starts at 1; https://github.com/mp3guy/ElasticFusion/blob/master/Core/src/ElasticFusion.cpp#L52. So we can safely use option 1.